app/pages/multisig/ProposalView/util.ts

```diff
@@ -144,8 +150,31 @@ async function HandleUpdateInstructionSignatureFile(
         };
     }
 
-    let validSignature = false;
-    validSignature = await isSignatureValid(proposal, signature);
+    let blockSummary;
+    try {
+        blockSummary = (await fetchLastFinalizedBlockSummary())
+            .lastFinalizedBlockSummary;
+    } catch {
+        return {
+            show: true,
+            header: errorMessages.unableToReachNode,
+            content:
+                'Unable to verify signature is signed by an authorized key without connection to node.',
+        };
+    }
+    const handler = findUpdateInstructionHandler(proposal.type);
+    try {
+        await attachKeyIndex(signature, blockSummary, proposal, handler);
```

Comment:
It is a little hacky to reuse `attachKeyIndex` for validation as you are not trying to attach the key index. Either create a new method with a `boolean` return type, or use `findKeyIndex()` directly and check that instead of `try/catch` on this method.